app/controllers/users_controller.rb

```diff
@@ -2,8 +2,23 @@ class UsersController < ApplicationController
   before_action :validate_session
   
   def show
-    mood = params[:mood]
     @user = current_user
-    @task = TasksFacade.new.task_by_mood(@user.id, mood)
+    @mood = params[:mood]
+    @task = TasksFacade.new.task_by_mood(@user.id, @mood) 
```

Comment:
Need to call the cookie in the task_by_mood method- else it will look like 
```ruby
TasksFacade.new.task_by_mood(current_user.id, nil)
```
if the user refreshes the page